File: Code/Flight.py
import Reservation


class Flight:
    def __init__(self, flight_id, departure, arrival,
                 departure_time, arrival_time, num_reservations):
        self.flight_id = flight_id
        self.departure_airport = departure
        self.arrival_airport = arrival
        self.departure_time = departure_time
        self.arrival_time = arrival_time
        self.duration = arrival_time - departure_time
        self.num_reservations = num_reservations
        self.reservations = [Reservation(self, None, reservation, 10)
                             for reservation in range(num_reservations)]

        def print_flight():
            pass

File: TBDAirline/TBDAirline/Models/Flight.cs
namespace TBDAirline.Models
{
    /// <summary>
    /// Flight contains all the important attributes related a flight.
    /// Flight is used in conjuntion with other data types to construct a reservation in Book
    /// </summary>
    public class Flight
    {
        public int FlightID { get; set; }
        public DateTime DepartureTime { get; set; }
        public decimal Price { get; set; }
        public int FromID {get; set; }
        public int ToID { get; set; }
        public string Duration { get; set; }
        public DateTime ArrivalTime { get; set; }

    }
}
